extensions/sun-moon-times/src/components/CityListItemView.tsx

```diff
@@ -0,0 +1,102 @@
+import { Action, ActionPanel, Icon, List, useNavigation } from "@raycast/api"
+import { getSunrise, getSunset } from "sunrise-sunset-js"
+import { countryList } from "../../assets/countryList"
+import { CityItem } from "../../types/CityItem"
+import { convertDateToString } from "../common/convertDateToString"
+import { getDayDuration } from "../common/getDayDuration"
+import { resolveCoords } from "../common/resolveCoords"
+import { DetailView } from "./DetailView"
+import { useFavorites } from "./FavoritesProvider"
+
+interface CityListItemViewProps extends CityItem {
+    isFavorite?: boolean
+}
+
+export const CityListItemView = ({
+    geonameId,
+    name,
+    countryCode,
+    timezone,
+    coordinates,
+    isFavorite,
+}: CityListItemViewProps) => {
+    const { push } = useNavigation()
+    const city = { geonameId, name, countryCode, timezone, coordinates }
+    const sunrise = getSunrise(coordinates.lat, coordinates.lon)
+    const sunset = getSunset(coordinates.lat, coordinates.lon)
+    const dayDuration = getDayDuration(sunrise, sunset)
+    const sunriseString = convertDateToString(sunrise, timezone)
+    const sunsetString = convertDateToString(sunset, timezone)
+
+    const { addToFavorites, removeFromFavorites, moveFavorite } = useFavorites()
+
+    return (
+        <List.Item
+            key={geonameId}
+            icon={{ source: countryList[countryCode].flag }}
+            title={name}
+            subtitle={resolveCoords(coordinates.lat, coordinates.lon)}
+            accessories={[
+                { icon: Icon.Sun },
+                {
+                    icon: Icon.ArrowUp,
+                    text: sunriseString,
+                },
+                {
+                    icon: Icon.ArrowDown,
+                    text: sunsetString,
+                },
+                { icon: Icon.Clock, text: dayDuration },
+            ]}
+            actions={
+                <ActionPanel>
+                    <Action
+                        title="Show More Info"
+                        icon={Icon.Info}
+                        onAction={() =>
+                            push(
```

Comment:
how about using a [List.Item.Detail](https://developers.raycast.com/api-reference/user-interface/list#list.item.detail) instead of `Detail` here? 🤔

Reply:
@sxn I changed the Detail Info View to List.Item.Detail. I think this was a great idea because now you have all info in one view.